devel/glasskube/Portfile

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+version             0.16.0
+github.setup        glasskube glasskube $version v
```

Comment:
please add "github.tarball_from" and set it to the appropriate value. If upstream provides them "releases", otherwise "archive".